jax_unirep/evotuning.py

```diff
@@ -29,14 +29,7 @@
     validate_mLSTM1900_params,
 )
 
-# setup logger
 logger = logging.getLogger("evotuning")
-logger.setLevel(logging.INFO)
-fh = logging.FileHandler("evotuning.log")
-fh.setLevel(logging.INFO)
-formatter = logging.Formatter("%(asctime)s :: %(levelname)s :: %(message)s")
-fh.setFormatter(formatter)
-logger.addHandler(fh)
```

Comment:
I've got a proposal for this block of code. What if we hid it inside a function block, and called it when executing evotuning? Proposal:

```python
def setup_evotuning_log():
    # code block gets moved inside here.
    # returns nothing
````

Then, in `fit`, right at the top:

```python
def fit(...):
    """Docstring goes here."""
    setup_evotuning_log()
```

Reply:
That would work great! I've updated the code to your proposal.